pkg/apis/apps/v1/zz_generated.validations.go

```diff
@@ -38,8 +38,7 @@ func init() { localSchemeBuilder.Register(RegisterValidations) }
 func RegisterValidations(scheme *runtime.Scheme) error {
 	scheme.AddValidationFunc(&v1.DaemonSet{}, func(obj, oldObj interface{}, subresources ...string) field.ErrorList {
 		if len(subresources) == 0 {
-			root := obj.(*v1.DaemonSet)
-			return Validate_DaemonSetSpec(&root.Spec, nil)
+			return Validate_DaemonSet(obj.(*v1.DaemonSet), nil)
```

Comment:
One perk of the previous way (explicitly Spec) is that the name was clear: it doesn't include status and it doesn't include metadata.  Is it better this way or the previous?

Reply:
ooh, good catch.  I glazed over this when spot checking.  I'll revert this behavior